### PR TITLE
Zwave inclusion refactor

### PR DESCRIPTION
**Pull request by cdjackson**
This refactors the initialisation sequences when a node is included into the network. Information from the AddNode message is used to bypass the initial stages which significantly improves the speed at which nodes complete initialisation once initially included into the network.

This should help speed up battery device inclusion as initialisation starts immediately, and also reduces delays which may contribute to failures in secure inclusion.

### CODE REVIEW

File: addons/binding/org.openhab.binding.zwave/src/main/java/org/openhab/binding/zwave/internal/protocol/serialmessage/AddNodeMessageClass.java [hunk 1]
```diff
@@ -67,9 +79,12 @@ public SerialMessage doRequestStop() {
         SerialMessage newMessage = new SerialMessage(SerialMessage.SerialMessageClass.AddNodeToNetwork,
                 SerialMessage.SerialMessageType.Request, SerialMessage.SerialMessageClass.AddNodeToNetwork,
                 SerialMessage.SerialMessagePriority.High);
-        byte[] newPayload = { (byte) ADD_NODE_STOP };
 
-        newMessage.setMessagePayload(newPayload);
+        ByteArrayOutputStream outputData = new ByteArrayOutputStream();
+        outputData.write(ADD_NODE_STOP);
+        outputData.write(255); // TODO: This should use the callbackId
```
Comment (jongj) [inline addons/binding/org.openhab.binding.zwave/src/main/java/org/openhab/binding/zwave/internal/protocol/serialmessage/AddNodeMessageClass.java]: In the original version this byte was not there. Is it a copy/paste error or really intended?

File: addons/binding/org.openhab.binding.zwave/src/main/java/org/openhab/binding/zwave/internal/protocol/serialmessage/RemoveNodeMessageClass.java [hunk 2]
```diff
@@ -55,9 +60,12 @@ public SerialMessage doRequestStop() {
         SerialMessage newMessage = new SerialMessage(SerialMessage.SerialMessageClass.RemoveNodeFromNetwork,
                 SerialMessage.SerialMessageType.Request, SerialMessage.SerialMessageClass.RemoveNodeFromNetwork,
                 SerialMessage.SerialMessagePriority.High);
-        byte[] newPayload = { (byte) REMOVE_NODE_STOP };
 
-        newMessage.setMessagePayload(newPayload);
+        ByteArrayOutputStream outputData = new ByteArrayOutputStream();
+        outputData.write(REMOVE_NODE_STOP);
+        outputData.write(255); // TODO: This should use the callbackId
```
Comment (jongj) [inline addons/binding/org.openhab.binding.zwave/src/main/java/org/openhab/binding/zwave/internal/protocol/serialmessage/RemoveNodeMessageClass.java]: In the original version this byte was not there. Is it a copy/paste error or really intended?

Comment (cdjackson) [inline addons/binding/org.openhab.binding.zwave/src/main/java/org/openhab/binding/zwave/internal/protocol/serialmessage/RemoveNodeMessageClass.java]: It’s intended - as per the comment, it’s the callback id and it fixes a bug.

File: addons/binding/org.openhab.binding.zwave/src/main/java/org/openhab/binding/zwave/internal/protocol/serialmessage/AddNodeMessageClass.java (hunk 1 above)
Comment (cdjackson) [inline addons/binding/org.openhab.binding.zwave/src/main/java/org/openhab/binding/zwave/internal/protocol/serialmessage/AddNodeMessageClass.java]: It’s intended - it’s the callback id and it fixes a bug with the incorrectly formatted packet.

**Pull request merged**